CMakeLists.txt

```diff
@@ -52,11 +49,20 @@ set(_src
   stempy/reader.cpp
   python/pyreader.cpp
   stempy/mask.cpp
-  stempy/electronthresholds.cpp
-  h5cpp/h5readwrite.cpp)
+  stempy/electronthresholds.cpp)
 
 set(_libs Threads::Threads)
-list(APPEND _libs "${HDF5_LIBRARIES}")
+list(APPEND _libs pybind11::headers)
+# We have to link to Python3::Module since "pyreader.cpp" includes
+# pybind11 and python header files.
+list(APPEND _libs Python3::Module)
```

Comment:
So the `stempy` library now depends directly on Python?

Reply:
Yes, just because `pyreader.cpp` is compiled as part of the source, and `pyreader.h` files are included.